changelog.json

```diff
@@ -1,5 +1,25 @@
 {
   "releases": {
+    "1.1.1-beta2": [
+      "[New] Render WebP images in diffs - #4164. Thanks @agisilaos!",
+      "[Fixed] Edit context menus in commit form input elements - #3886",
+      "[Fixed] Escape behavior should match that of Branch List - #3972",
+      "[Fixed] Keep caret position after inserting completion - #3835. Thanks @CarlRosell!",
+      "[Fixed] Handle error events when watching files used to get Git LFS output - #4117",
+      "[Fixed] Potential race condition when opening a fork pull request - #4149",
+      "[Fixed] Show placeholder image when no pull requests found - #3973",
+      "[Fixed] Disable input fields summary and description while commit in progress - #3893. Thanks @crea7or!",
+      "[Fixed] Ensure pull request cache is cleared after last pull request merged - #4122",
+      "[Fixed] Focus two-factor authentication dialog on input - #4220. Thanks @WaleedAshraf!",
+      "[Improved] Add `Discard All Changes...` to context menu on changed file list - #4197. Thanks @xamm!",
+      "[Improved] Improve contrast for button labels in app toolbar - #4219",
+      "[Improved] Speed up check for submodules when discarding - #4186. Thanks @kmscode!",
```

Comment:
This is specifically for Windows right?

Reply:
The issue was reported for Windows, but the fix is available to everyone